doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb

```diff
@@ -0,0 +1,1075 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "plt.scatter(training_systems[\"energies\"], predict[\"energies\"])\n",
+    "x_range = np.linspace(\n",
+    "    np.min(training_systems[\"energies\"]) * 0.999,\n",
+    "    np.max(training_systems[\"energies\"]) * 1.001,\n",
+    "    20,\n",
+    ")\n",
+    "plt.plot(x_range, x_range, \"r--\", linewidth=0.25)\n",
+    "plt.xlabel(\"Energy of DFT\")\n",
+    "plt.ylabel(\"Energy predicted by deep potential\")\n",
+    "plt.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fffaad23-53a6-47dd-8d91-9beb68d1212b",
+   "metadata": {},
+   "source": [
+    "## Run MD with LAMMPS\n",
+    "\n",
+    "The model can drive molecular dynamics in LAMMPS. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "d0ceac75-9753-4ae8-b954-c2cc6d005e46",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "conf.lmp  graph.pb  in.lammps\n"
+     ]
+    }
+   ],
+   "source": [
+    "! cd ../02.lmp && cp ../01.train/graph.pb ./ && ls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c794572f-7e9c-42f7-b78a-4d82bd3f3118",
+   "metadata": {},
+   "source": [
+    "Here `conf.lmp` gives the initial configuration of a gas phase methane MD simulation, and the file `in.lammps` is the lammps input script. One may check in.lammps and finds that it is a rather standard LAMMPS input file for a MD simulation, with only two exception lines:\n",
+    "```\n",
+    "pair_style  deepmd graph.pb\n",
+    "pair_coeff  * *\n",
+    "```\n",
+    "where the pair style deepmd is invoked and the model file `graph.pb` is provided, which means the atomic interaction will be computed by the DP model that is stored in the file `graph.pb`. \n",
+    "\n",
+    "In an environment with a compatibable version of LAMMPS, the deep potential molecular dynamics can be performed via \n",
+    "\n",
+    "```bash\n",
+    "lmp -i input.lammps\n",
```

Comment:
Is there any reason not to perform it?

Reply:
The `bohrium notebook` image used by this notebook does not have Lammps installed.

Reply:
It is not too hard to install one, using `pip install --force-reinstall deepmd-kit[gpu,cu11,lmp]`